test/type/self.swift

```diff
@@ -25,3 +25,19 @@ extension X {
 extension X.Inner {
   func foo(_ other: Self) { } // expected-error{{'Self' is only available in a protocol or as the result of a method in a class; did you mean 'X.Inner'?}}{{21-25=X.Inner}}
 }
+
+// SR-695
+class Mario {
+  func getFriend() -> Self { return self; } // expected-note{{overridden declaration is here}}
```

Comment:
The semicolon is not needed here

Reply:
Yep! In my defense, it's copy-pasted from the original issue sample code.